Web/starter_project_vanguard/components/blog/FileUpload.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import Image from 'next/image'
+
+
+const FileUpload = () => {
+  return (
+    <div className="w-full mb-16 h-1/2   lg:h-2/3  mt-8 bg-gray-100 rounded-lg flex items-center justify-center flex-col p-2 lg:p-5">
+
+      <Image src='/images/illustration.jpg' alt="illustration image" width={200} height={200}/>
```

Comment:
i think it is better if all imports aren't written inline here. import the above the component.